templates/settings.html

```diff
@@ -4,6 +4,15 @@
 
 <h3>Settings</h3>
 
+<h4>Opt-out of Leaderboard</h4>
+<form method="get">
```

Comment:
Just curious, any reason why this one's a get?
And is it possible to opt back in once you've opted out?

Reply:
I saw an example of it online and wanted to try it out. It should probably be a post, right?

Reply:
> It should probably be a post, right?

Uhh......I'm not actually sure, that's why I figured I'd ask you 😅. Just more used to seeing POSTs